provisioner/terraform/resources.go

```diff
@@ -464,7 +470,31 @@ func ConvertResourcesAndParameters(modules []*tfjson.StateModule, rawGraph strin
 		}
 	}
 
-	return resources, parameters, nil
+	// A map is used to ensure we don't have duplicates!
+	gitAuthProvidersMap := map[string]struct{}{}
+	for _, tfResources := range tfResourcesByLabel {
+		for _, resource := range tfResources {
+			//
+			if resource.Type != "coder_git_auth" {
```

Comment:
Should we remove git auth providers from resources?

Reply:
This is a data-source anyways, so it can't be represented with an agent or anything.